### ISSUE TITLE
App crashes after update to 8.1.0-rc.1

### ISSUE THREAD

**Issue by luke-hawk** [NativeScript/ns-v8ios-runtime#125]
Hi, after upgrading to `8.1.0-rc.1` we've seen a lot of app crashes on ios. It seems like these crashes are caused by forcefully closing the app (double tap on home button / swipe up). On testflight we even get an alert that the app has crashed after closing it that way. Here's a stacktrace from crashlytics:

```
Crashed: com.apple.main-thread
0  NativeScript                   0x101c0ff2c v8::base::OS::Abort() + 24
1  NativeScript                   0x101403758 v8::Isolate::Dispose() + 158
2  NativeScript                   0x101403758 v8::Isolate::Dispose() + 158
3  NativeScript                   0x10136f1dc tns::Runtime::~Runtime() + 164
4  NativeScript                   0x1012dc1cc std::__1::shared_ptr<v8::Persistent<v8::Value, v8::NonCopyablePersistentTraits<v8::Value> > >::~shared_ptr() + 64
5  libsystem_c.dylib              0x1a80dfcf8 __cxa_finalize_ranges + 372
6  libsystem_c.dylib              0x1a80e0008 exit + 24
7  UIKitCore                      0x1a1d7789c -[UIApplication terminateWithSuccess] + 498
8  UIKitCore                      0x1a1442f6c -[_UISceneLifecycleMultiplexer _evalTransitionToSettings:fromSettings:forceExit:withTransitionStore:] + 124
9  UIKitCore                      0x1a1442bf0 -[_UISceneLifecycleMultiplexer forceExitWithTransitionContext:scene:] + 216
10 UIKitCore                      0x1a1d6d7d4 -[UIApplication workspaceShouldExit:withTransitionContext:] + 208
11 FrontBoardServices             0x1adf9e51c -[FBSUIApplicationWorkspaceShim workspaceShouldExit:withTransitionContext:] + 84
12 FrontBoardServices             0x1adfc8c44 __63-[FBSWorkspaceScenesClient willTerminateWithTransitionContext:]_block_invoke_2 + 76
13 FrontBoardServices             0x1adfaf420 -[FBSWorkspace _calloutQueue_executeCalloutFromSource:withBlock:] + 232
14 FrontBoardServices             0x1adfc8be0 __63-[FBSWorkspaceScenesClient willTerminateWithTransitionContext:]_block_invoke + 104
15 libdispatch.dylib              0x19f0ff298 _dispatch_client_callout + 16
16 libdispatch.dylib              0x19f0d7ff8 _dispatch_block_invoke_direct$VARIANT$armv81 + 216
17 FrontBoardServices             0x1adfede04 __FBSSERIALQUEUE_IS_CALLING_OUT_TO_A_BLOCK__ + 40
18 FrontBoardServices             0x1adfedacc -[FBSSerialQueue _targetQueue_performNextIfPossible] + 404
19 FrontBoardServices             0x1adfedfa0 -[FBSSerialQueue _performNextFromRunLoopSource] + 28
20 CoreFoundation                 0x19f4468a8 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 24
21 CoreFoundation                 0x19f4467a8 __CFRunLoopDoSource0 + 204
22 CoreFoundation                 0x19f445afc __CFRunLoopDoSources0 + 256
23 CoreFoundation                 0x19f440018 __CFRunLoopRun + 768
24 CoreFoundation                 0x19f43f7d0 CFRunLoopRunSpecific + 572
25 GraphicsServices               0x1b5b81570 GSEventRunModal + 160
26 UIKitCore                      0x1a1d6c2d0 -[UIApplication _run] + 1052
27 UIKitCore                      0x1a1d7184c UIApplicationMain + 164
28 NativeScript                   0x1013e0044 ffi_call_SYSV + 68
29 NativeScript                   0x1013df01c ffi_call_int + 968
30 NativeScript                   0x101381734 tns::Interop::CallFunctionInternal(tns::MethodCall&) + 428
31 NativeScript                   0x101343a98 std::__1::__function::__func<tns::MetadataBuilder::CFunctionCallback(v8::FunctionCallbackInfo<v8::Value> const&)::$_2, std::__1::allocator<tns::MetadataBuilder::CFunctionCallback(v8::FunctionCallbackInfo<v8::Value> const&)::$_2>, void ()>::operator()() + 564
32 NativeScript                   0x1013a4cac tns::Tasks::Drain() + 104
33 NativeScript                   0x10139eecc +[NativeScript start:] + 572
34 dermanosticmobile              0x1006d6270 main + 100 (main.m:100)
35 libdyld.dylib                  0x19f11e140 start + 4
```

Let me know if you need any more information to get to the root of the issue or if I can do anything to help solving it :)

**Comment by rigor789** [comment]
I believe this has been fixed in 8.1.0 now.

**Comment by luke-hawk** [comment]
@rigor789 Yes, I can confirm it has been fixed in 8.1.0. Thanks a lot 👍
